Report.md

```diff
@@ -96,8 +96,43 @@ Design Thinking Approach
 
 
 ## **Agile Methodology - Scrum**
-4. Explain the agile methodology, namely, Scrum you have used to carry out the procedure. In your explanation, ensure that you outline activities from Scrum that you use.
+> 4. Explain the agile methodology, namely, Scrum you have used to carry out the procedure. In your explanation, ensure that you outline activities from Scrum that you use.
 
+Scrum is the most widely used agile methodology for software design and development. The framework assumes that requirements and the development process are constantly and unpredictably changing which is often the case in real-world scenarios. While traditional models/approaches (e.g. Waterfall, Spiral) attempt to define all requirements upfront, requirements in agile/scrum are developed, reviewed, and iteratively updated to allow for greater adaptability to changes in client needs/feedback, technological advancements, and other environmental factors (e.g. global pandemics).
+
+### Roles
+Scrum teams are self-organising and cross-functional meaning they are independent and trusted to be internally directed. Authority is given to the team to complete work in a manner that the team sees best fit. A number of roles exist in most scrum teams. Due to the nature of this project, the product owner and scrum master also act as members of the development team.
+
+#### Product Owner – Zac
+The product owner is an individual responsible for establishing and communicating the product vision including defining and prioritising features to be developed. The product owner must make quick and informed decisions while monitoring project progress and release dates.
+#### Scrum Master – Aiswarya
+The scrum master facilitates scrum team events and promotes scrum theory, rules, and practices.
+#### Development Team - Anastasia, Sanya, Sean
+The development team is responsible for building the deliverable increment of a product in each sprint.
+
+### Events/Activities
+
+A sprint is a fixed period of time after which a deliveralble iteration of a product must be developed. The following activities were conducted in each of our team's 1-week sprints (Sunday to Friday).
```

Comment:
```suggestion
A sprint is a fixed period of time after which a deliverable iteration of a product must be developed. The following activities were conducted in each of our team's 1-week sprints (Sunday to Friday).
```

Reply:
There's an empty line here. Looking at the actual markdown I can't tell the difference so on second thought it's probably no big deal at all - just OCD more than anything